src/gui/windows/download_controller_mappings.zig

```diff
@@ -0,0 +1,45 @@
+const std = @import("std");
+
+const main = @import("root");
+const files = main.files;
+const settings = main.settings;
+const Vec2f = main.vec.Vec2f;
+
+const gui = @import("../gui.zig");
+const GuiComponent = gui.GuiComponent;
+const GuiWindow = gui.GuiWindow;
+const Button = @import("../components/Button.zig");
+const CheckBox = @import("../components/CheckBox.zig");
+const Label = @import("../components/Label.zig");
+const VerticalList = @import("../components/VerticalList.zig");
+const HorizontalList = @import("../components/HorizontalList.zig");
+
+pub var window = GuiWindow {
+	.contentSize = Vec2f{128, 64},
+	.pos = Vec2f{0, 0},
```

Comment:
.pos is intended for internal usage only and is overwritten frequently, instead you should init the .relativePosition.